### DISCUSSION TITLE
Who's Using and How?

### DISCUSSION THREAD

**Discussion by yoavniran** [rpldy/react-uploady#131] [Show and tell]
# Using Uploady in Production?

## Or are you planning to?

Please share your use-case and company (if relevant).
It will be great to know how Uploady is used in the wild 🧐

**Comment by dbousamra** [comment]
We are using it in production: 
![image](https://user-images.githubusercontent.com/648137/100722812-bec89380-340c-11eb-851d-7c441457c00e.png)

Have had no major issues.

**Comment by yoavniran** [comment]
That's awesome @dbousamra 
Who's "we"? 🙂

**Comment by oferitz** [comment]
We at [Duality](https://dualitytech.com/) using it. mostly with chunk-uploady to upload very large data files. 
It's an awesome library, thanks for creating it ❤️

**Comment by smartlightning** [comment]
We [(Capgemini) ](https://www.capgemini.com/) are using it for media upload for the federal police in a state in south Germany. I've got a lot of answers to all my questions. Thanks so much for your effort!

**Reply by yoavniran** [reply]
very cool @smartlightning! thanks for sharing

**Comment by trb** [comment]
I'm using it at [goodfact](https://goodfact.co) to handle uploading lots of files/big files. Target audience isn't always very tech-savvy and after trying a bunch of different components/libraries I felt that Uploady gave me enough control to build an interface with good feedback on progress/status without requiring me to write a ton of manual logic. Great library, one of my favorites, the flexibility vs. functionality tradeoff is spot-on.

**Reply by yoavniran** [reply]
hey @trb this is awesome! Thanks for sharing. Always love to learn about the use-cases Uploady helps support

**Comment by AJLemos** [comment]
We love using Uploady at SmugMug! It's a fantastic library.

<img width="1820" height="1529" alt="CleanShot 2025-11-10 at 09 26 49" src="https://github.com/user-attachments/assets/6cb24173-d2c4-4168-92d3-062d9c44af9f" />
